<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>网络聊天室</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#3465a4;&quot;&gt;网络聊天室&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QLineEdit" name="groupText"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portText"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ipText"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="groupLabel">
        <property name="text">
         <string>聊天组</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="emailText"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>服务器IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>服务器端口</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="nameText"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>聊天名称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ipText</tabstop>
  <tabstop>portText</tabstop>
  <tabstop>emailText</tabstop>
  <tabstop>nameText</tabstop>
  <tabstop>groupText</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
